image/image_linux.go

```diff
@@ -417,6 +419,21 @@ func setupSeed(tsto *ToolingStore, model *asserts.Model, opts *Options) error {
 		if err := installCloudConfig(rootDir, gadgetUnpackDir); err != nil {
 			return err
 		}
+
+		gadgetInfo, err := gadget.ReadInfo(gadgetUnpackDir, model)
+		if err != nil {
+			return err
+		}
+
+		defaultsDir := filepath.Join(rootDir, "_writable_defaults")
```

Comment:
we might want a  helper in sysconfig,   sysconfig.WritableDefaultsDir(rootdir,  subdir...) that does:

filepath.Join(filepath.Join(rootdir, "_writable_defaults"), subdir...)

it can be used also in @mvo5 new code (#8612) etc